testing/path.go

```diff
@@ -162,6 +162,20 @@ func (path *Path) SetupClients() {
 	}
 }
 
+// SetupEurekaClients is a helper function to create clients supporting ibc-eureka on both
+// chains. It assumes the caller does not anticipate any errors.
+func (path *Path) SetupEurekaClients() {
```

Comment:
Rather than doing this all in one step, let's keep SetupClients unchanged, and **add** a new function called `SetupCounterparties` that does the provide counterparty for each side.

Then in our tests, we can setup like so:

```go
path.SetupClients()
path.SetupCounterparties()
```

Reply:
eventually `path.Setup()` will just do both

Reply:
created a SetupV2 that does both